transactionoutbox-acceptance/src/test/java/com/gruelbox/transactionoutbox/acceptance/TestComplexConfigurationExample.java

```diff
@@ -47,7 +48,8 @@ void test() {
                 DefaultPersistor.builder()
                     // Selecting the right SQL dialect ensures that features such as SKIP LOCKED are
                     // used correctly.
-                    .dialect(Dialect.POSTGRESQL_9)
+                    .dialect(dialect)
+                    .sequenceGenerator(DefaultSequenceGenerator.builder().dialect(dialect).build())
```

Comment:
This shouldn't be necessary (see comments on `DefaultPersistor`)

Reply:
fixed